src/main/kotlin/com/pubnub/api/models/consumer/access_manager/v3/PNRevokeTokenResult.kt

```diff
@@ -0,0 +1,3 @@
+package com.pubnub.api.models.consumer.access_manager.v3
+
+data class PNRevokeTokenResult(val token: String)
```

Comment:
It would be good to include what we have to return to the user in case of successful call in the specification document so we're consistent across the SDKs